CHANGELOG.rst

```diff
@@ -33,6 +33,64 @@ and this project adheres to
 .. Attribution
 .. ^^^^^^^^^^^
 
+[v2.0.0] - 2023-02-03
+~~~~~~~~~~~~~~~~~~~~~
+
+Summary
+^^^^^^^
+
+* This major release migrates X-PSI from Python2 (X-PSI v1.2.1 or lower) to Python3 (, with corresponding updates and improvements to all documentation and tutorials.  
```

Comment:
Is that open bracket supposed to be there?

Reply:
Not so much :-D